.travis.yml

```diff
@@ -5,13 +5,27 @@ language: node_js
 node_js:
   - 'stable'
 
+env:
+  # Keep star at top so it runs first - it'll be the longest task
+  - PROJECT="*"
```

Comment:
Is there still a reason to leave in `*`? As long as all the individual projects are also listed here, I'm not sure that it adds any value, and we'll be running everything twice.